### ISSUE TITLE
Big Sur Update instructions

### ISSUE THREAD

**Issue by stschulze1972** [zearp/Nucintosh#27]
hey zearp,

thanks for your great work. this isn't an issue, but i dont know how to transport my question ...

**Please insert any update Instructions to the Nucintosh-wiki....how to update from 11.1 -> 11.2.**

Thanks and regards

**Comment by zearp** [comment]
There are no instructions to give really. You can update by using ```Software Update``` inside macOS. Only rarely would you need to update the EFI. If it fails to install updating the EFI may help, unless the update is an early beta and/or introduces issues that need to be fixed in OpenCore or a kext.

I've updated my test machine from 11.1 beta to 11.2 with no issues so it should be fine. But it's always a good idea to create/update a bootable backup before doing any OS upgrades, just so it's easy to roll back in case it does break something. Updating within a major release, like from 11.1 to 11.2 usually doesn't require any EFI changes.

If an update does introduce things that break Hackintoshes its usually fixed within a week or so. So if you don't want to make a backup and try it out yourself its best to wait a while before updating. You can check Hackintosh forums and see what people are saying about the update.

tl;dr Just make a backup then update and see if it works. If Apple introduces something that breaks things simply roll back and wait for fixes upstream or try and sort it out yourself (part of the fun for me).

PS: If you rely on macOS --daily driver-- I would suggest sticking with Catalina. It won't receive any updates that could break things but will still receive at least a few years of security and Safari updates. Catalina has been very reliable stable here.

**Comment by stschulze1972** [comment]
Thanks for your answer. Update is perfect working.

regards
